crates/bevy_log/src/lib.rs

```diff
@@ -134,7 +134,68 @@ pub(crate) struct FlushGuard(SyncCell<tracing_chrome::FlushGuard>);
 ///         .run();
 /// }
 /// ```
+/// # Example Setup
 ///
+/// For a quick setup that enables all first-party logging, while not showing any of your dependencies' log data
+/// you can configure the plugin as below.
+///
+/// ```no_run
+/// fn main() {
+///     App::new()
+///         .add_plugins(DefaultPlugins.set(LogPlugin {
+///             filter: "warn,my_crate=trace".to_string(), //specific filters
+///             level: Level::TRACE,//Change this to be globally change levels
+///             ..Default::default()
+///         }))
+///         .add_systems(Startup, setup)
+///         .run();
+/// }
+/// ```
+/// The filter (in this case an EnvFilter) chooses whether to print the log,the most specific filters apply
+/// in the above case default is `warn` but a more specific case for `my_crate`
+/// is to use `trace`
```

Comment:
I'm not sure what this sentence means, so I'm not sure how to revise it.

Reply:
I think maybe an example will work if we start with a filter like this code `filter: "warn".to_string(),`
It says to only print logs at warning level or above.

Now say we add a second clause
`filter: "warn,my_crate=trace".to_string()`
warn is the default level to print but if we in `my_crate` then we print at trace level because that is the more specific level.

Reply:
Okay, let's see what that looks like :)

Reply:
Ok have had another go at it.